src/shims/foreign_items/posix.rs

```diff
@@ -255,25 +255,12 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 this.write_null(dest)?;
             }
 
-            // Incomplete shims that we "stub out" just to get pre-main initialziation code to work.
```

Comment:
Oops, that's a typo we have 4 times...

Reply:
Would you like me to fix it in the other three foreign_items files here, or should that be saved for outside this PR?

Reply:
If you are fixing one, please also fix the others.